src/app/public/modules/progress-indicator/progress-indicator.component.ts

```diff
@@ -170,6 +170,15 @@ export class SkyProgressIndicatorComponent implements OnInit, AfterContentInit,
 
     this.updateSteps();
 
+    this.itemComponents
+      .changes
+      .takeUntil(this.ngUnsubscribe)
+      .subscribe(() => {
+        this.updateSteps();
+        this.changeDetector.markForCheck();
+        this.notifyChange();
```

Comment:
This will cause `progressChanges` to emit. Our docs state that `progressChanges`...
> Fires when the progress indicator changes the status of a progress item.

Do you think # of steps is the same thing as the "status" changing?

Reply:
Yup. There is a list of statuses that is included in that output. When an item is updated we add that new item to the status array. I feel like this is correct as the user will need to know about the new item's status.

Reply:
Good argument. Sounds good to me 👍